build/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent none
+
+    environment {
+        MATTERMOST_DESKTOP_WINDOWS_PFX_PASSWORD = credentials('mattermost-desktop-windows-pfx-password')
+    }
+    stages {
+        stage('Build Linux') {
+            agent any
+            steps {
+                echo 'Building Mattermost Desktop App'
+                sh '''#!/bin/bash
```

Comment:
I think that `set -e` is needed for each bash script in order to stop the pipeline as script error when an error occurred in a certain command.